From 3dd3af7b7bb3e3ae661e4d6c24b9f68ecb0a0a56 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v2-02.11-3dd3af7b7bb-20221119T125550Z-avarab@gmail.com>
In-Reply-To: <cover-v2-00.11-00000000000-20221119T125550Z-avarab@gmail.com>
References: <cover-00.12-00000000000-20221118T112205Z-avarab@gmail.com>
	<cover-v2-00.11-00000000000-20221119T125550Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Wed, 2 Nov 2022 10:07:45 +0100
Subject: [PATCH v2 02/11] builtin/{grep,log}.: don't define
 "USE_THE_INDEX_COMPATIBILITY_MACROS"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Adding "USE_THE_INDEX_COMPATIBILITY_MACROS" to these two appears to
have been unnecessary from the start, as going back and compiling
f8adbec9fea (cache.h: flip NO_THE_INDEX_COMPATIBILITY_MACROS switch,
2019-01-24) without that addition works.

Let's not have these ask for the compatibility macros from cache.h
that they don't need.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/grep.c | 1 -
 builtin/log.c  | 1 -
 2 files changed, 2 deletions(-)

diff --git a/builtin/grep.c b/builtin/grep.c
index 5fa927d4e22..f7821c5fbba 100644
--- a/builtin/grep.c
+++ b/builtin/grep.c
@@ -3,7 +3,6 @@
  *
  * Copyright (c) 2006 Junio C Hamano
  */
-#define USE_THE_INDEX_COMPATIBILITY_MACROS
 #include "cache.h"
 #include "repository.h"
 #include "config.h"
diff --git a/builtin/log.c b/builtin/log.c
index 5eafcf26b49..89447a50838 100644
--- a/builtin/log.c
+++ b/builtin/log.c
@@ -4,7 +4,6 @@
  * (C) Copyright 2006 Linus Torvalds
  *		 2006 Junio Hamano
  */
-#define USE_THE_INDEX_COMPATIBILITY_MACROS
 #include "cache.h"
 #include "config.h"
 #include "refs.h"
-- 
2.38.0.1509.g9445af83948

